src/chart/sankey/SankeyView.ts

```diff
@@ -220,6 +222,15 @@ class SankeyView extends ChartView {
                     }
             }
 
+            setLabelStyle(
+                curve, getLabelStatesModels(edgeModel, 'edgeLabel'),
+                {
+                    labelFetcher: seriesModel,
+                    labelDataIndex: edge.dataIndex,
+                    defaultText: `${edgeModel.get('value')}`
```

Comment:
template string is unnecessary here.

Reply:
`get` function returns a `number`, but `defaultText` is a `string`, using template to get rid of ts error